README.md

```diff
@@ -44,6 +44,32 @@ $ ./injectived.sh
 
 Voila! You have now successfully setup a full node on the Injective Chain.
 
+### Install for MacOS
+1. Build from source
+   ```bash
+   git clone https://github.com/OpenDeFiFoundation/injective-core.git
+   cd injective-core
+   make install
+   ```
+
+2. update ``.zshrc`` - Add these lines to your ``~/.zshrc`` file:
+   ```bash
+   export GOPATH=$HOME/go
+   export PATH=$PATH:$(go env GOPATH)/bin
+   export GOPRIVATE=github.com/InjectiveLabs/injective-core
```

Comment:
this line is definitely not required as it points to a different repo

Reply:
please remove `export GOPRIVATE=github.com/InjectiveLabs/injective-core` this line